tgui/packages/tgui/interfaces/Aquarium.tsx

```diff
@@ -17,9 +24,29 @@ type Data = {
   minTemperature: number;
   maxTemperature: number;
   fluidTypes: string[];
-  contents: { ref: string; name: string }[];
+  fish_data: fish_data[];
+  prop_data: prop_data[];
   allow_breeding: BooleanLike;
   feeding_interval: number;
+  heart_icon: string;
+  heart_icon_state: string;
+  heart_empty_icon_state: string;
+};
+
+type fish_data = {
+  fish_ref: string;
+  fish_name: string;
+  fish_happiness: number;
+  fish_icon: string;
+  fish_icon_state: string;
+  fish_health: number;
+};
+
+type prop_data = {
+  prop_ref: string;
+  prop_name: string;
+  prop_icon: string;
+  prop_icon_state: string;
 };
 
 export const Aquarium = (props) => {
```

Comment:
this component is massive, and should be broken up into more parts. components are ideally 30-100 lines each, and files are no larger than 500